main/tables.py

```diff
@@ -103,6 +105,10 @@ class Meta:
             "class": "table table-striped table-hover table-responsive",
         }
 
+    def render_cost_centre(self, value: str, record: Funding) -> str:
+        """Render cost centre and activity code together."""
+        return f"{value}_{record.activity}"
+
 
```

Comment:
I have done it this way but I could've also created a project_code property which may be simpler? The column could still then be ordered using `order_by=("cost_centre", "activity")`